lesson_36-HW41/src/store/index.js

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit';
+import routReducer from './routSlice';
```

Comment:
```suggestion
import rootReducer from './rootSlice';
```

Напевно ви хотіли скзати кореневий редюсер)
слово `rout` має зовсім інше значення
краще виправте скрізь